lib/chef/resource/powershell_package_source.rb

```diff
@@ -62,105 +156,167 @@ class PowershellPackageSource < Chef::Resource
         else
           status = repo.result
         end
-        url status["url"]
+        source_name status["source_name"]
```

Comment:
the deprecation of url to source_name seems to be sort of half done and its not clear where the url can still be passed to a deprecated property

Reply:
corrected with an alias modifier.